### PR TITLE
Use GithubActions and add return types

### PR DESCRIPTION
**Pull request by franmomu**
The result of the Github Actions can be seen in https://github.com/franmomu/CraueFormFlowBundle/pull/1

This also fixes some deprecations shown using Symfony 6.3 (adding return types)

### CODE REVIEW

**Comment by franmomu** [comment]
friendly ping @craue if you can review this

**Comment by craue** [comment]
First of all, thanks for all your work. I really appreciate it, @franmomu.

I don't have much experience with GitHub Actions yet, only started playing around for CraueConfigBundle a few months ago. And of course I'd prefer a similar setup in all my repos in order to simplify maintainability. So if you're interested in helping me to achieve this by
- pointing me to what could/should be improved for the existing setup in CraueConfigBundle and
- finishing this PR,
that would be great!

Furthermore, I guess it would be better to put everything related to return types into a separate PR to focus on the GHA migration in this one.

Let me know what you think.

**Comment by craue** [comment]
I finally made some progress and used most of your commits in #417. I'd appreciate your feedback.

It became pretty clear that, while reworking CI, trying to get rid of deprecations also means using your commit about adding return types so that #416 is no longer needed. Sorry for the back & forth.

File: Tests/AppKernel.php [hunk 1]
```diff
@@ -36,33 +36,28 @@ public function registerBundles() : iterable {
...
+	{
+		return __DIR__;
 	}
 
-	public function unserialize($data) {
-		list($environment, $configFile) = unserialize($data);
-		$this->__construct($environment, $configFile);
+	private function getBaseDir(): string
+	{
+		return sprintf('%s/craue-form-flow-bundle/var/', sys_get_temp_dir());
```
Comment (craue) [inline Tests/AppKernel.php:61]: What's the reason for this change? Was it meant to simplify things?
In fact, I still need these env vars while running tests locally to put temporary files on a RAM drive. 🤓

Comment (franmomu) [reply Tests/AppKernel.php:61]: Yeah, I think it was to remove the other methods, so if you need the env vars, keep them

**Comment by franmomu** [comment]
closed in favor of https://github.com/craue/CraueFormFlowBundle/pull/417
